src/app/modals/aji-intercept/aji.component.ts

```diff
@@ -31,16 +32,38 @@ export class AJIInterceptModal implements OnInit {
   closeModal: EventEmitter<number> = new EventEmitter();
 
   constructor(
-    private _auth: AuthService,
+    private _auth: AuthService, private _account: AccountService
   ) { }
 
   ngOnInit() { }
 
+  public updateUser(updateUser) {
+    this._account.update(updateUser).pipe(
+      take(1),
+      map(res => {
+        this._auth.saveUser(updateUser)
+      },
+        err => {
+          console.error(err)
+        }
+      )).subscribe()
+  }
+
   public dismissModal(): void {
     this._auth.store('AJIInterceptClosed', true);
     this.closeModal.emit()
   }
 
+  public remindMeLater(): void {
```

Comment:
If the user clicks remind me later we don't want to do the following. This should only happen if they dimiss the modal by clicking X.

Reply:
If they click remind me later we just want to close the modal for this session.